go.mod

```diff
@@ -2,8 +2,10 @@ module github.com/aws-controllers-k8s/elasticache-controller
 
 go 1.14
 
+replace github.com/aws-controllers-k8s/runtime => ../runtime
```

Comment:
I don't think we should keep the `replace` statement here, it should be used only for dev purposes. If the `runtime` repository doesn't exist locally, `go build` will complain with `...runtime/go.mod: no such file or directory`. Other than that there is also the risk of using an outdated version/commit of `runtime`.

Reply:
@A-Hilaly is correct on the above.

Reply:
Updated the go.mod file.